app/controllers/frontend_controller.rb

```diff
@@ -110,7 +110,7 @@ def pick_buyer_or_provider_layout
   end
 
   def all_services
-    @api_selector_services = current_user.accessible_services.to_a if current_account.try!(:provider?)
+    @api_selector_services = current_user.accessible_services.includes(:account).to_a if current_account.try!(:provider?)
```

Comment:
For this:
https://github.com/3scale/porta/blob/52782b2eb40a2cc877e87f6257d32b040f5f3257/app/views/shared/_api_selector.html.slim#L7
And:
https://github.com/3scale/porta/blob/52782b2eb40a2cc877e87f6257d32b040f5f3257/app/models/service.rb#L397

Reply:
By the way why do we do `to_a` here?

Reply:
Absolutely no idea 😂 😝 But you reviewed it in https://github.com/3scale/porta/pull/31/files#diff-1862ca4b36d9904f13fb697bb4daab9eR112 and when I saw your comment, I assumed you understood it 😂 
Otherwise... @thomasmaas explain it for us plz 🤔 😄

Reply:
OK well no need of the `to_a` I suppose.

Reply:
And there is this also https://github.com/3scale/porta/pull/31/files#diff-96675bcac39ce105d4fe0aaaaaaa5de6R9
It should use `api_selector_services.any?` instead of size, because `size` will do a `COUNT(*)` in the database 
Stale review :/